### PR TITLE
VEN-835 | Add loading spinner to BerthPage and WinterStoragePage contents

### PR DESCRIPTION
**Pull request by ahie**
## Description :sparkles:

- Adds loading spinners to content pages.
- Adds text support for Spinner component.

### Closes :no_good_woman:

[VEN-835](https://helsinkisolutionoffice.atlassian.net/browse/VEN-835)

### Automated tests :gear:️

- `src/components/common/spinner/Spinner.test.tsx`

### Manual testing :construction_worker_man:

- Open either the berth (`${locale}/berths`) or winter storage page (`${locale}/winter-storage`)
- A spinner should be present for both Map and List tabs, including loading text, while the data is still being fetched. Note: might have to simulate higher network latency to test loading time.

### CODE REVIEW

File: src/components/common/spinner/spinner.scss [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
-  border: 2px solid $hel-light;
-  border-top: 2px solid $hel-black;
-  border-radius: 50%;
-  width: 72px;
-  height: 72px;
-  animation: spin 1s linear infinite;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1em;
```
Comment (Nizar-Rahme) [inline src/components/common/spinner/spinner.scss:7]: Can we use predefined spacing values from here: `src/assets/styles/_variables.scss`?

File: src/components/common/spinner/spinner.scss [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
+    border: 2px solid $hel-light;
+    border-top: 2px solid $hel-black;
+    border-radius: 50%;
+    width: 72px;
+    height: 72px;
+    animation: spin 1s linear infinite;
+  }
+
+  &__text {
+    margin-top: 0.5em;
```
Comment (Nizar-Rahme) [inline src/components/common/spinner/spinner.scss:19]: And here ^

**Review by Nizar-Rahme — CHANGES_REQUESTED** [review]
I left a couple of comments, otherwise, looks good!

File: src/components/common/spinner/spinner.scss (hunk 2 above)
Comment (ahie) [reply src/components/common/spinner/spinner.scss]: Done.

File: src/components/common/spinner/spinner.scss (hunk 1 above)
Comment (ahie) [reply src/components/common/spinner/spinner.scss]: Done.

**Review by TatuArvela — APPROVED** [review]
Nothing to add, looks good to me! 👍

**Review by Nizar-Rahme — APPROVED** [review]
👍

**Pull request merged**